src/components/CurrencySelector/CurrencySelector.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react'
+import { StyledDropdown } from './styles'
+
+export default class CurrencySelector extends Component {
+
+  state = {
+    currency: "usd"
+  }
+
+  handleSelect = (e) => {
+    this.setState({ currency: e.target.value })
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+    if (this.state.currency !== prevState.currency){
+      this.setState({currency: this.state.currency})
```

Comment:
what's the purpose of this, state.currency is already state.currency

Reply:
ah right, got rid of the setState